.github/pr_management/common.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * Collection of common functions to use in this folder
+ */
+import * as core from '@actions/core'
+import * as github from '@actions/github';
+const token = core.getInput("repo-token");
+const octokit = github.getOctokit(token);
+
+const owner = github.context.repo.owner;
+const repo = github.context.repo.repo;
+const issue_number = github.context.issue.number;
+const actor = github.context.actor;
+
+//// variables to configure
+// todo change in teammates
+const usualTimeForChecksToRun = 5000; // 20 * 60 * 1000; // min * sec * ms
+
+// to prevent cyclical checking when checking for passing runs. note: needs to match names assigned by workflow files
+const excludedChecksNames = {
+    "Handle PR that may be draft": 1,
```

Comment:
These magic strings should be turned into const at the beginning of the file, so changing them in the future is easier

Reply:
These strings are only used in one place in the js files, and they correspond to the action names in the yml files. Should I still abstract them into const?

Reply:
@pPris I feel like its better to do so, the benefit of doing so might not be too visible now, but will be in the future